src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/ServiceConfigItem.java

```diff
@@ -62,5 +74,26 @@ public void setAllowCreatingCaseBeforePaymentsAreProcessed(boolean allowCreating
         this.allowCreatingCaseBeforePaymentsAreProcessed = allowCreatingCaseBeforePaymentsAreProcessed;
     }
 
+    public String getSurnameMapping(String formType) {
+        return surnameMappings.get(formType);
+    }
+
+    public void setSurnameMappings(List<FormFieldMapping> surnameMappings) {
+        this.surnameMappings = surnameMappings.stream()
+            .collect(groupingBy(FormFieldMapping::getFormType, toList()))
+            .entrySet().stream()
+            .collect(toMap(
+                e -> e.getKey(),
+                e -> {
+                    if (e.getValue().size() > 1) {
+                        log.error("Form type {} has {} mappings to surname fields",
+                            e.getKey(),
+                            e.getValue().size());
```

Comment:
are you sure we should proceed here?

Reply:
This is a requirement: if field appears more than once, log an error, but use the first value found

Reply:
should we log every time it gets used, or just once, on application startup though?

Reply:
I believe it is done right

Reply:
The app should fail if we have multiple mapping for the same jurisdiction + form type. The error log is about finding multiple such keys in the actual data.